tests/test_genomes.py

```diff
@@ -10,146 +10,222 @@
+                assert np.all(a1.shape == a2.shape)
+                assert np.all(a1 == a2)
+            self.verify_dfe_breakpoints(contig)
+
+    @pytest.mark.skip(reason="TODO allow more flexible inputs")
+    def test_add_DFE_interval_formats(self):
+        L = 50818
+        for intervals in (
+            [[0, L]],
+            [[0, int(0.2 * L)]],
+            ([0, int(0.2 * L)], [int(0.6 * L), L]),
+        ):
+            contig = stdpopsim.Contig.basic_contig(length=L)
+            contig.add_DFE(intervals=intervals, DFE=self.example_dfe)
+            np.testing.assert_array_equal(intervals, contig.interval_list[1])
 
-    def test_is_neutral3(self):
-        contig = stdpopsim.Contig.basic_contig(length=100)
-        contig.clear_DFEs()
-        props = [0.3, 0.7]
-        mt = [
-            stdpopsim.ext.MutationType(distribution_type="e", distribution_args=[1])
-            for _ in props
-        ]
-        dfes = [
-            stdpopsim.DFE(
-                id=str(j),
-                description="test",
-                long_description="test test",
-                proportions=props,
-                mutation_types=mt,
-            )
-            for j in range(2)
-        ]
-        contig.add_DFE(
-            intervals=np.array([[10, 30], [50, 100]]), DFE=dfes[0], fill_neutral=False
-        )
-        contig.add_DFE(intervals=np.array([[30, 40]]), DFE=dfes[1], fill_neutral=False)
-        assert not contig.is_neutral
+    def test_is_neutral(self):
+        for neutral in (True, False):
+            for dist in ("f", "e"):
+                contig = stdpopsim.Contig.basic_contig(length=100)
+                contig.clear_DFEs()
+                props = [0.3, 0.7]
+                if neutral:
+                    s = 0
+                else:
+                    s = 0.1
+                mt = [
+                    stdpopsim.MutationType(
+                        distribution_type=dist, distribution_args=[s]
+                    )
+                    for _ in props
+                ]
+                dfes = [
+                    stdpopsim.DFE(
+                        id=str(j),
+                        description="test",
+                        long_description="test test",
+                        proportions=props,
+                        mutation_types=mt,
+                    )
+                    for j in range(2)
+                ]
+                contig.add_DFE(
+                    intervals=np.array([[10, 30], [50, 100]]),
+                    DFE=dfes[0],
+                )
+                contig.add_DFE(intervals=np.array([[30, 40]]), DFE=dfes[1])
+                # exponential with mean zero doesn't count as neutral!
+                assert contig.is_neutral is (neutral and dist == "f")
+
+
+class TestMutationTypes(object):
+    def test_netural_mutation_type(self):
+        mt = stdpopsim.MutationType()
+        assert mt.is_neutral
+        assert mt.is_neutral()
```

Comment:
Tests are failing here.
```suggestion
```

Reply:
Gah, whoops.  This is because I made `MutationType.is_neutral` a property, btw, to match `Contig.is_neutral`. I think both should be the same, whichever it is; I'm not sure which is better. Any reason not to have them properties?

Reply:
I personally don't use properties much, but this is a fairly cheap operation so no objections!